e2e/src/utils/retries.js

```diff
@@ -6,6 +6,8 @@ export const launchApp = async (
     permissions: { notifications: 'YES', contacts: 'YES', camera: 'YES' },
     launchArgs: {
       detoxPrintBusyIdleResources: 'YES',
+      // Use new tx feed from Zerion by default
+      statsigGateOverrides: 'show_zerion_transaction_feed=true',
```

Comment:
this doesn't guarantee all e2e tests use this as a launch arg right? Some tests seem to set an override for the launchArgs param without the statsigOverride, is it ok if those tests don't use the zerion tx feed? We can also make it so that this override is always set in this function, if we set this override always inside the function

Reply:
Fixed in 87179f5d0f6d76078e1b84dbe3f865e1a943daf2; The custom `launchApp` will now use the `defaultLaunchArgs` and allow specific properties to be overwritten while maintaining the rest of the defaults.

Reply:
@satish-ravi Opted to show the old TX feed as switching the e2e tests to the new TX feed is outside of the ticket's scope and resulted in many additional changes.